docs/setup.md

```diff
@@ -30,25 +30,33 @@ Follow this guide to setup your development machine.
     ```
     cp settings/dev.sample.py settings/dev.py
     ```
+
+6. Creating user
```

Comment:
It should be Creating postgres user ?
We need to put point why they need this step.